lib/smart_answer_flows/marriage-abroad/outcomes/countries/thailand/_opposite_sex.erb

```diff
@@ -40,52 +42,32 @@ You'll be told if you need to provide any other documents. For example, if you
 
-###What to bring to your appointment
+[Download an application pack](/government/publications/information-for-british-nationals-intending-to-marry-in-thailand) and follow the instructions to fill it in.
 
-You need to bring several documents to your appointment. If any of your documents were issued outside the UK or Thailand, they must be ‘legalised’ (certified as genuine) by the [Ministry of Foreign Affairs in Thailand](http://www.mfa.go.th/main/en/services/16265-Naturalization-Legalization.html).
+Do not sign the affirmation - you’ll need to sign it later at the embassy when you swear your affirmation.
 
-If your affirmation or documents are not in English, you’ll need to bring [certified translations](/government/publications/thailand-list-of-lawyers) into Thai as well as the originals.
+##Book an appointment at the British embassy in Bangkok
 
-You’ll need to bring:
+You need to book an appointment to swear your affirmation at the British embassy in Bangkok. 
 
-- your passport
-- your application pack with the affirmation, if you applied using the paper form
-
-If you’ve changed your name by deed poll, you also need to bring evidence of your change of name.
-
-If you’ve been married or in a civil partnership before, you’ll also need:
-
-- your original [decree absolute or final order](/copy-decree-absolute-final-order) (this could be a PDF and covering email from the court) - if you’re divorced
-- your annulment certificate - if your marriage or civil partnership was annulled
-- your civil partnership dissolution - if your civil partnership was dissolved
-- your partner’s [death certificate and your marriage certificate](/order-copy-birth-death-marriage-certificate)  - if you’ve been widowed
-
-If your divorce, civil partnership dissolution or annulment took place outside the UK, you’ll need evidence that you or your ex partner lived in or were a national of that country at the time.
+You'll be told how to book the appointment when you apply for the affirmation online or by paper form. You'll also be told what documents you must bring to your appointment, including your affirmation.
 
 ##Translate and legalise your affirmation
 
-You’ll need to get your affirmation [translated](/government/publications/thailand-list-of-lawyers) into Thai and ‘legalised’ (certified as genuine) by the [Ministry of Foreign Affairs in Thailand](http://www.mfa.go.th/main/en/services/16265-Naturalization-Legalization.html).
+After your appointment at the embassy, you need to get your affirmation both:
+
+- [translated into Thai](/government/publications/thailand-list-of-lawyers)
+- ‘legalised’ (certified as genuine) by the [Ministry of Foreign Affairs in Thailand](http://www.mfa.go.th/main/en/services/16265-Naturalization-Legalization.html)
```

Comment:
Can I check if this URL is correct?  I get an error when I visit the page.

![Screenshot 2021-07-19 at 14 18 55](https://user-images.githubusercontent.com/6329861/126166159-17a2dfc4-7ae1-4145-af15-e2daba9b9f21.png)